src/app/admin/components/filter-search/filter-search.component.ts

```diff
@@ -0,0 +1,168 @@
+import {
+    Component,
+    OnInit,
+    OnDestroy,
+    ViewChild,
+    ElementRef,
+    Output,
+    EventEmitter
+  } from '@angular/core';
+  import { Subscription } from 'rxjs/Subscription';
+  import { Observable } from 'rxjs/Rx';
+  import 'rxjs/add/observable/fromEvent';
+  import 'rxjs/add/operator/map';
+  import { CollectionService } from 'app/core/collection.service';
+  import { Router } from '@angular/router';
+  import { AuthService } from 'app/core/auth.service';
+  import { Subject } from 'rxjs';
+import { ContextMenuService } from 'app/shared/contextmenu/contextmenu.service';
+import { Placeholder } from '@angular/compiler/src/i18n/i18n_ast';
+import { Key } from 'protractor';
+// import { EventEmitter } from 'protractor';
+
+@Component({
+  selector: 'clark-admin-filter-search',
+  templateUrl: './filter-search.component.html',
+  styleUrls: ['./filter-search.component.scss']
+})
+export class FilterSearchComponent implements OnInit, OnDestroy {
+    private subscriptions: Subscription[] = [];
+    private searchText$: Observable<string>;
+    selectedCollection: { abvName: string, name: string };
+    selectedStatusFilter: string[];
+    collections: { abvName: string, name: string }[] = [];
+    isCollectionRestricted = false;
+    filtersModified$: Subject<void> = new Subject();
+    // collection
+    // filters applied to dashboard objects (status filters)
+    filters: Map<string, boolean> = new Map();
+
+    @Output() statusFilter = new EventEmitter<string>();
+    @Output() collectionFilter = new EventEmitter<string>();
+    @ViewChild('searchInput') searchInput: ElementRef;
+
+    searchText;
+    filterMenu: string;
+    filterMenuDown = false;
+    collectionMenu: string;
+    collectionMenuDown = false;
+    collectionFilterOptions: string[];
+    statusFilterOptions: string[];
+    constructor(
+      private collectionService: CollectionService,
+      private authService: AuthService,
+      private contextMenuService: ContextMenuService,
+      private router: Router) { }
+
+    ngOnInit() {
+      this.subToCollections();
+      this.findUserRestrictions();
+    }
+
+    private subToCollections(): void {
```

Comment:
It looks like we are fetching collections here. We should use this information to populate the collection menu. Make sure that all functions in this file are being used and provide JSDoc for each.